### PR TITLE
Wt 338 resend timer

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Resend timer
- TODO: The actual resending is not valid at this point

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-338 #done

### CODE REVIEW

**Comment by rebelchris** [comment]
@sshanzel Converted this one to useTimer as well 🙏

File: packages/shared/src/components/auth/AuthOptions.tsx [hunk 1]
```diff
@@ -158,7 +161,10 @@ function AuthOptions({
             title="Verify your email address"
             onClose={onClose}
           />
-          <EmailVerificationSent email={email} />
+          <EmailVerificationSent
+            email={email}
+            onResend={() => onRegister(registrationValues)}
+          />
```
Comment (sshanzel) [inline packages/shared/src/components/auth/AuthOptions.tsx:167]: I think resending the registration might result in an error as the user already exists. We might start a verification flow. Though I'm not sure tho about the current response.

**Review by sshanzel — COMMENTED** [review]
Just had one concern regarding the resending.

File: packages/shared/src/components/auth/AuthOptions.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/components/auth/AuthOptions.tsx:167]: I guess we can also start the verification flow, was unsure at the time what was the needed flow.

Comment (rebelchris) [reply packages/shared/src/components/auth/AuthOptions.tsx]: @sshanzel Added the verification flow now.
Can't test the emails, but response from Kratos is "email_sent"

Screenshot 2022-08-17 at 15 17 13

Comment (sshanzel) [reply packages/shared/src/components/auth/AuthOptions.tsx]: Oh, nice. Thought it was the registration flow getting utilized. Nice. Then this is perfect.

**Review by sshanzel — APPROVED** [review]
Let's go! 🚢

**Pull request merged**